util.go

```diff
@@ -4,10 +4,15 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net/http"
+	"os"
 )
 
 func httpGet(url string) (string, error) {
-	resp, err := http.Get(url)
+
+	req, _ := http.NewRequest("GET", url, nil)
+	req.Header.Set("Authorization", "Bearer "+os.Getenv("QUAYIO_TOKEN"))
```

Comment:
Please pass API token as method argument,

```go
func httpGet(url, apiToken string) (string, error) {
```

and not to attach `Authorization` header if API token is empty.

```go
	if apiToken != "" {
		req.Header.Set("Authorization", "Bearer "+apiToken)
	}
```